service/src/test/java/uk/gov/hmcts/reform/fpl/service/ApplicantServiceTest.java

```diff
@@ -130,4 +137,45 @@ void shouldNotAddNewPartyIdWhenApplicantsAlreadyHasPartyIdValue() {
 
         assertThat(applicant.getParty().partyId).isEqualTo(uuid);
     }
+
+    @Test
+    void shouldReturnApplicantCollectionWithOrganisationDetailsWhenOrganisationExists() {
+        CaseData caseData = CaseData.builder().build();
+
+        List<Applicant> applicants = unwrapElements(service.expandApplicantCollection(
+            caseData, buildOrganisation()));
+
+        assertThat(applicants.get(0).getParty().getOrganisationName())
+            .isEqualTo(buildOrganisation().getName());
+        assertThat(applicants.get(0).getParty().getAddress()).isEqualTo(buildOrganisation()
+            .getContactInformation().get(0).toAddress());
+    }
+
+    @Test
+    void shouldReturnApplicantCollectionWithoutOrganisationDetailsWhenNoOrganisationExists() {
+        CaseData caseData = CaseData.builder().build();
+
+        List<Applicant> applicants = unwrapElements(service.expandApplicantCollection(
+            caseData, Organisation.builder().build()));
+        assertThat(applicants.get(0).getParty().getOrganisationName()).isNull();
```

Comment:
lets have a space inbetween applicants and the assertion

Reply:
cool :)